packages/appium/docs/en/cli/env-vars.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Server Environment Variables
+---
+
+The primary way of configuring the Appium server is via [CLI Args](./args.md). However, some more
+advanced features are toggled or configured via environment variables. To set environment
+variables, refer to the documentation for your operating system and terminal. These are the
+environment variables that the Appium server understands:
+
+|Variable&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
```

Comment:
Are these non-breaking spaces needed? Other doc files don't seem to have them...

Reply:
without them, the env vars are wrapped mid-token